pypyr/cli.py

```diff
@@ -35,6 +35,9 @@ def get_parser():
                         help='Integer log level. Defaults to 20 (INFO). '
                         '10=DEBUG\n20=INFO\n30=WARNING\n40=ERROR\n50=CRITICAL'
                         '.\n Log Level < 10 gives full traceback on errors.')
+    parser.add_argument('--logfile', dest='log_file', type=str,
+                        default='pypyr.log',
```

Comment:
remove default entirely - as written this means pypyr would always write to pypyr.log, it would never not write to disk. in a lot of contexts pypyr might run in, it might well not have write access to disk. it should be end-user choice whether to write to disk or not.

Removing this line entirely means log_file will default to `None`, which is what we want. . .

Reply:
Also this. . .